.github/renovate.json

```diff
@@ -17,24 +17,12 @@
       "matchManagers": [
         "dockerfile"
       ],
-      "excludePackageNames": [
-        "ubuntu"
-      ],
       "matchUpdateTypes": [
         "major",
         "minor",
         "patch"
       ],
       "enabled": false
-    },
-    {
-      "matchManagers": [
-        "dockerfile"
-      ],
-      "packageNames": [
-        "ubuntu"
-      ],
-      "versioning": "regex:^(?<compatibility>[a-z]+)-(?<patch>\\d+)$"
```

Comment:
❓ Could you help me understand why this is no longer relevant? I understand that we removed the version so the `versioning` is irrelevant. But why the rest?

Reply:
We were telling Renovate not to ignore ubuntu for the normal Docker updates, and instead use a custom regex matcher. Since we're moving back to using a normal tag that Renovate can handle, we can let it update Ubuntu again.